rmarkdowns/Week_2.Rmd

```diff
@@ -160,66 +197,131 @@ Now let's add the lakes and rivers to our map.
 
 **Challenge 4: Update the `coord_sf` with your bounding box and your `ggtitle` with your own title.** The `states` layer should also now be updated with your new `states` dataframe you created in Challenge 2. 
 
+# Amber's Challenge 4 Answer
+
+I wanted to add railroads, too. Just editing your existing code chunk.
+
 ```{r}
+# lakes
+lakes50 <- ne_download(
+  scale = 50, 
+  type = "lakes", 
+  category = "physical")
+
+# rivers
+rivers50 <- ne_download(
+  scale = 50,
+  type = "rivers_lake_centerlines",
+  category = "physical",
+  returnclass = "sf"
+)
+
+# railroads
+railroads10 <- ne_download(
+  scale = 10, # only available in scale10, so let's see what that's about
```

Comment:
So Natural Earth has data available at 1:10m, 1:50m, and 1:110 million scales, so for more detailed data in smaller areas, you use the "10" scale data, which is only available in the rnaturalearthhires (high resolution) package :)